<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="recentBtn">
            <property name="text">
             <string>Recent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="starredBtn">
            <property name="text">
             <string>Starred</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="homeBtn">
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="desktopBtn">
            <property name="text">
             <string>Desktop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="documentsBtn">
            <property name="text">
             <string>Documents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadsBtn">
            <property name="text">
             <string>Downloads</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="musicBtn">
            <property name="text">
             <string>Music</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="picturesBtn">
            <property name="text">
             <string>Pictures</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="videosBtn">
            <property name="text">
             <string>Videos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="trashBtn">
            <property name="text">
             <string>Trash</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <widget class="QListWidget" name="listWidget"/>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
